saleor/graphql/schema.graphql

```diff
@@ -910,6 +910,8 @@ input FulfillmentUpdateTrackingInput {
 
 enum GatewaysEnum {
   DUMMY
+  BRAINTREE
```

Comment:
I think we have only DUMMY enabled by default, do the other come from your local settings?

Reply:
Oh, my bad!